src/bin/GraphOptions.ts

```diff
@@ -0,0 +1,65 @@
+import Integer from '../Integer';
+import ChainInfo from '../Node/ChainInfo';
+import GraphDescription from "../Graph/GraphDescription";
+
+// These defaults will be used if the relevant option is not given on the command line.
+const DEFAULT_RPC_PORT = 60000;
+const DEFAULT_WS_PORT = 8101;
+const DEFAULT_RPC_ADMIN_PORT = 8020;
+const DEFAULT_IPFS_PORT = 5001;
+const DEFAULT_POSTGRES_PORT = 5432;
```

Comment:
This will result in a conflicting port.
Created to a ticket to address this.
https://github.com/OpenST/mosaic-chains/issues/97